src/api/config/routes/webui_routes.rb

```diff
@@ -428,4 +428,6 @@
       end
     end
   end
+
+  resources :reports, except: [:new], controller: 'webui/reports'
```

Comment:
You also don't have any other of the default actions. You `only: [:create]`

Reply:
Right. We won't be using other routes for reports. This is updated